.github/pull_request_template.md

```diff
@@ -8,6 +8,6 @@
 
 ---
 #### Ship list
-- [ ] Run migration steps in readme in /atd-vzd/ if needed
+- [ ] Check migrations to make sure they work with latest in master branch
```

Comment:
Wording? Basically, I'm trying to say that we need to check that the migrations are in the correct order. Just like we do in Moped.

Reply:
this works but how about
Check migrations for any conflicts with latest migrations in master branch

Reply:
🙏 Yes, i like that a lot better! I'm updating.